src/include/sof/schedule/ll_schedule_domain.h

```diff
@@ -41,6 +41,7 @@ struct ll_schedule_domain_ops {
 	void (*domain_clear)(struct ll_schedule_domain *domain);
 	bool (*domain_is_pending)(struct ll_schedule_domain *domain,
 				  struct task *task, struct comp_dev **comp);
+	void (*domain_task_cancel)(struct ll_schedule_domain *domain, uint32_t num_tasks);
```

Comment:
This should be early in the seriess, right? I mean I think the previous patch already used this, so this would break bisect.

Reply:
You're right, this should definitely be added before the introduction of the preempt domain, thanks!